cvat/apps/annotation/README.md

```diff
@@ -150,3 +150,173 @@ It allows to download and upload annotations in different formats and easily add
+
+- supported shapes: Rectangles
+- limitations: Support of Pascal VOC object detection format
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+#### How to create a task from Pascal VOC dataset
+1. Download the Pascal Voc dataset
+1. Create the CVAT task with the following labels:
+   ```
+   aeroplane bicycle bird boat bottle bus car cat chair cow diningtable dog horse motorbike person pottedplant sheep sofa train tvmonitor
+   ```
+   You can add `~checkbox=difficult:false ~checkbox=truncated:false` attributes if you want to use it.
+
+   Select interesting image files (See [Creating an annotation task](cvat/apps/documentation/user_guide.md#creating-an-annotation-task) guide for details)
+1. zip the corresponding annotation files
+1. click `Upload annotation` button, choose `Pascal VOC ZIP 1.0` and select the *.zip file with annotations from previous step. It may take some time.
+
+### [YOLO](https://pjreddie.com/darknet/yolo/)
+#### Dumper description
+- downloaded file: a zip archive with following structure:
+  ```
+  taskname.zip
+  ├── frame000001.txt
+  ├── frame000002.txt
+  ├── ...
+  └── obj.names
+  ```
+  Each `*.txt` file contains annotations for a frame with the same name as annotation file. Short description of  `*.txt` file structure: each line describes label and bounding box in the following format `label_id cx cy w h`.
+  `obj.names` contains the ordered list of label names.
+- supported shapes - Rectangles
+
+#### Loader description
+- uploaded file: a zip archive with following structure:
+  ```
+  taskname.zip
+  ├── frame000001.txt
+  ├── frame000002.txt
+  ├── frame000003.txt
+  ├── ...
+  └──obj.names
+  ```
+  Each `*.txt` file contains annotations for a frame with the same name as annotation file. Note: it should be possible to match the CVAT frame(imagename) and annotation filename. There are 2 options:
+  1. full match between image name and filename form annotation `*.txt` file (in case of a task was created from images or archive of images).
+  2. match by frame number (if CVAT cannot match by name). If name of the *.txt annotation file contains a number CVAT will interpret it as frame number.
+
+- supported shapes: Rectangles
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+### [MS COCO Object Detection](http://cocodataset.org/#format-data)
+#### Dumper description
+- downloaded file: single unpacked `json`. Detailed description of the MS COCO format can be found [here](http://cocodataset.org/#format-data)
+- supported shapes - Polygons, Rectangles (interpreted as polygons)
+
+#### Loader description
+- uploaded file: single unpacked `*.json`.
+- supported shapes: Polygons
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+#### How to create a task from PMS COCO dataset
```

Comment:
PMS -> MS. Could you please look at the whole text one more time and fix obvious typos?